template.yml

```diff
@@ -70,3 +97,10 @@ Resources:
                   - "CreateStack"
                   - "UpdateStack"
                   - "DeleteStack"
+
+  MonitoringDashboard:
```

Comment:
trenger du denne? dette vil vel fort medføre at Dashboard blir nullstilt på deploy og ikke satt igjen før en kjører oppdatering?

Reply:
For å slette dashbordet hvis stacken slettes.